polkadot/node/subsystem-util/src/runtime/mod.rs

```diff
@@ -135,7 +136,7 @@ impl RuntimeInfo {
 	/// Create with more elaborate configuration options.
 	pub fn new_with_config(cfg: Config) -> Self {
 		Self {
-			session_index_cache: LruMap::new(ByLength::new(cfg.session_cache_lru_size.max(10))),
+			session_index_cache: LruMap::new(ByLength::new(cfg.session_cache_lru_size.max(2 * MAX_FINALITY_LAG))),
```

Comment:
Adding a comment/argument why this is a good value would be good. (E.g. just copying your PR description)

Reply:
Will do.